static/js/components/PersonalForm.js

```diff
@@ -74,34 +79,17 @@ export default class PersonalForm extends ProfileFormFields {
               {...this.defaultInputComponentProps()}
             />
           </Cell>
+          <Cell col={12}>
+            {this.boundGeosuggest(addressMapping, "Current address",
+              {
+                id: "current-home",
+                placeholder: "Example: 100 Main Street, Anytown, 01234, United States",
+                types: ["geocode"]
```

Comment:
[According to Google's documentation,](https://developers.google.com/maps/documentation/javascript/reference#AutocompletionRequest) `geocode` is the right value to use for addresses.